From 8042159c1487494cf8d0bc263bb66b92d819a38b Mon Sep 17 00:00:00 2001
From: linmin <linmin@eswincomputing.com>
Date: Tue, 2 Jul 2024 18:58:49 +0800
Subject: [PATCH 098/109] WIN2030-15279:chore(Makefile):Set DEBUG = 0

Changelogs::
1.Set "DEBUG = 0" in Makefile, otherwise the output elf file
  may differ from PC to PC.

Signed-off-by: linmin <linmin@eswincomputing.com>
Change-Id: Icd76e004889846e2f351bc7444a9e86802ffc792
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index b4181c3..19e47d0 100644
--- a/Makefile
+++ b/Makefile
@@ -26,7 +26,7 @@ TARGET = STM32F407VET6_BMC
 # building variables
 ######################################
 # debug build?
-DEBUG = 1
+DEBUG = 0
 # optimization
 OPT = -Og
 # OPT = -O2
-- 
2.25.1

